_source/_posts/2018-05-22-simple-multifactor-authentication-in-node.md

```diff
@@ -13,49 +13,58 @@ tweets:
 - "Who knew how easy adding multi-factor authentication to your #Node app is! #mfa #javascript"
 - "Learn how to quickly add multi-factor authentication to your #Node app! #mfa #javascript"
 type: conversion
+changelog:
+  - 2022-01-5: Updated dependencies, code, and screenshots. See this post's changes in [okta-blog#1017](https://github.com/oktadev/okta-blog/pull/1017).
```

Comment:
I think this needs to be 2022-01-05 for the changelog to render without quotes around the text.

Reply:
Gotcha. I've opened a PR here: https://github.com/oktadev/okta-blog/pull/1032